style.css

```diff
@@ -413,14 +422,260 @@ textarea::placeholder {
   opacity: 0.5;
 }
 
+.headerlist {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
 
+/* desktop version */
+
 @media (min-width: 768px) {
+  .icon-Union {
+    display: none;
+  }
+
+  .topBar-nav {
+    position: sticky;
+    top: 0;
+    background-color: white;
+  }
+
+  .logo {
+    padding-left: 100px;
+  }
+
+  .headerlist {
+    display: flex;
+    gap: 8px;
+    padding-right: 100px;
+    position: sticky;
+    top: 0;
+    height: 20px;
+    list-style-type: none;
+  }
+
+  .headerlist a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+  }
+
+  .headerlist a:hover {
+    color: rgb(24, 155, 160);
+  }
+
   .content-section {
+    padding-top: 50px;
+    background-image: url(img/Header\ bg-desktop.png);
+    background-size: contain;
+    display: flex;
+    height: 718px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border-bottom-left-radius: 80px 80px;
+  }
+
+  .main-Container {
+    width: 60%;
+    margin-left: 0;
+  }
+
+  .headingTxt {
+    color: #091e42;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    width: 400px;
+  }
+
+  .text-2 {
+    padding: 10px 0;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .work-section {
+    display: grid;
+    grid-template-columns: 1fr;
+    border-radius: 16px;
+    background-color: #f7f7f9;
+    align-items: center;
+    justify-content: center;
+    padding-top: 100px;
+    gap: 40px;
+    margin-bottom: 50px;
+    width: auto;
+    height: auto;
+    padding-bottom: 150px;
+  }
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the changes :clap: However the space at the left and the right of the projects are smaller compare to the Figma Design. Kindly check it to show your ability to parse a Figma design to create a UI.
**HIT: Use this CSS code for the `.work-section`**
```sh
.work-section {
    display: grid;
    grid-template-columns: 1fr;
    border-radius: 16px;
    align-items: center;
    justify-content: center;
    gap: 70px;
    margin-bottom: 50px;
    width: auto;
    height: auto;
    padding: 100px 5%;
}
```

### Your design :point_down: 
![Screenshot from 2023-05-06 13-32-27](https://user-images.githubusercontent.com/91142432/236627458-a287c812-49c9-4e64-9dec-10b73b0965fa.png)

### What it is supposed to look like :point_down: 
![Screenshot from 2023-04-06 10-25-14](https://user-images.githubusercontent.com/91142432/236627539-cdf00d38-76c8-432f-af21-fdcd6fbef3f2.png)

_This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change_